templates/rapidsai-core/Base.dockerfile.j2

```diff
@@ -25,7 +25,7 @@ RUN if [ "${BUILD_BRANCH}" = "main" ]; then sed -i '/nightly/d;/dask\/label\/dev
 ENV RAPIDS_DIR=/rapids
 
 {# Copy files needed by RAPIDS and 3rd parties for builds, test, and runtime. #}
-RUN mkdir -p ${RAPIDS_DIR}/utils {{ "${GCC9_DIR}/lib64" if "centos" in os }}
+RUN mkdir -p ${RAPIDS_DIR}/utils {{ "${GCC9_DIR}/lib64" if ( "rocky" in os or "centos" in os ) }}
```

Comment:
```suggestion
RUN mkdir -p ${RAPIDS_DIR}/utils
```

Since the switch to `conda` compilers, this conditional statement is no longer needed and can therefore be removed.

Reply:
removed.